docs/faq.md

```diff
@@ -0,0 +1,77 @@
+
+
+### Is there a central list of all hooks and tasks available in Tool Kit?
+
+Funny you should ask! We have just the thing:
+
+| Hook              | Hook exported by plugin | Possible tasks    |
+|-------------------|-------------------------|--------------------|
+| `run:local`         | npm                     | `Node`               |
+|                   |                         | `Nodemon`            |
+|                   |                         | `NextRouter`         |
+|                   |                         | `WebpackDevelopment` |
+|                   |                         | `WebpackWatch`       |
+| `test:local`        | npm                     | `Mocha`              |
+|                   |                         | `JestLocal`          |
+|                   |                         | `Eslint`             |
+|                   |                         | `SecretSquirrel`     |
+|                   |                         | `CypressLocal`     |
+| `build:local`      | npm                     | `BabelDevelopment`   |
+|                   |                         | `WebpackDevelopment` |
+| `build:ci`          | circleci                | `BabelProduction`    |
+|                   |                         | `WebpackProduction`  |
+| `test:ci`           | circleci                | `Eslint`             |
+|                   |                         | `Mocha`              |
+|                   |                         | `JestCi`             |
+|                   |                         | `CypressCi`             |
+| `deploy:review`     | circleci-heroku         | `HerokuReview`       |
+| `deploy:staging`    | circleci-heroku         | `HerokuStaging`      |
+| `test:review`       | circleci-heroku         | `NTest`              |
+|                   |                         | `Pa11y`              |
+| `test:staging`      | circleci-heroku         | `NTest`              |
+| `teardown:staging`  | circleci-heroku         | `HerokuTeardown`     |
+| `deploy:production` | circleci-heroku         | `HerokuProduction`   |
+| `publish:tag`       | circleci-npm            | `NpmPublish`         |
+| `cleanup:remote`    | heroku                  | `NpmPrune`           |
+| `release:remote`    | heroku                  | `UploadAssetsToS3`   |
+| `build:remote`      | heroku                  | `BabelProduction`    |
+|                   |                         | `WebpackProduction`  |
+| `git:precommit`     | husky-npm               | `LintStaged`         |
+|                   |                         | `SecretSquirrel`     |
+| `git:commitmsg`     | husky-npm               |                    |
+| `test:staged`       | lint-staged-npm         | `Eslint`             |
+| `format:staged`     | lint-staged-npm         | `Prettier`           |
+| `format:local`      | prettier                | `Prettier`           |
```

Comment:
**nitpick:** Can we format this table to have even spacing? Your IDE should be able to format it for you.